angular/projects/spark-angular/src/lib/components/sprk-autocomplete/sprk-autocomplete.stories.ts

```diff
@@ -125,7 +125,7 @@ export const defaultStory = () => ({
           id="autocomplete-input"
           name="autocomplete-input"
           type="text"
-          class="sprk-b-TextInput--has-svg-icon sprk-u-Width-100"
```

Comment:
Confirmed that this is removed because Katie's React Input PR will take care of adding width 100 to --has-svg class